assets/src/domain/eventEditor/services/context/TestContext/useDomTestData.ts

```diff
@@ -1,21 +1,25 @@
 import { useEffect } from 'react';
 import { event, currentUser, generalSettings } from './data';
+import type { EventEditorDomData } from '../../../interfaces/types';
 
-import { mockEeJsData } from '../../../../../application/services/config/test/data';
+import { mockEspressoDomData } from '../../../../../application/services/config/test/data';
 
 const useDomTestData = (): void => {
 	// Set the DOM data
-	window.eeEditorData = { event, currentUser, generalSettings };
-	window.eejsdata = { data: mockEeJsData };
+	const eventEspressoData: EventEditorDomData = {
+		...mockEspressoDomData,
+		eventEditor: { event },
+	};
+	eventEspressoData.config.currentUser = currentUser;
+	eventEspressoData.config.generalSettings = generalSettings;
```

Comment:
You forgot to add that assignment to `window.eventEspressoData`

Reply:
:facepalm: